dydxprotocol/v4-web
# feat: add buys and sells as marks on tv chart

This PR adds [Marks](https://www.tradingview.com/charting-library-docs/latest/ui_elements/Marks/) to the TradingView chart for historical buy and sell orders.

<img width="723" alt="image" src="https://github.com/user-attachments/assets/281f032c-d6a7-4e75-8747-df87a75b645e">
<img width="728" alt="image" src="https://github.com/user-attachments/assets/70e4ddb4-09ba-4632-b285-8c47b9131dfb">

- Adding `supports_marks: true` to the datafeed configuration will tell TV to query the new `getMarks` method
- Implemented `getMarks`
- Also did some minor cleanup in our custom css file by combining a bunch of duplicate toggle classnames into singular `.toggle` and `.toggle-active`


## Testing plan
- Create a bunch of orders and close them (or view a market for which you have a bunch of historic orders)
- Verify that the buy/sell marks show up on the chart for the correct time interval
- Verify that the text in the tooltip shows the correct size and average price
